src/GraphUpscale.cpp

```diff
@@ -23,22 +23,25 @@
 namespace smoothg
 {
 
+/// @todo why is there not timing here?
```

Comment:
Good catch, the timing is not consistent between the constructors.  It should be moved outside of the `Init` function, or move the `chrono.Stop();` block in the coarsening constructor up before the `Init` call (lines 62-63 moved up to 61-62).